packages/breadcrumbs/src/containers/BreadcrumbContainer.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class BreadcrumbContainer extends Component {
+  static propTypes = {
+    /**
+     * @param {Object} renderProps
+     * @param {Function} renderProps.getContainerProps - Props to be spread onto containing element
+     */
+    children: PropTypes.func,
+    /**
+     * Identical to children
+     */
+    render: PropTypes.func
+  };
+
+  getContainerProps = ({ role = 'navigation', ...other } = {}) => {
+    return {
+      role,
+      'aria-label': 'Breadcrumb navigation',
```

Comment:
> Should we default this like role in the argument destructuring to make it more obvious to teams that they'll need to translate this?

Not sure how clean this would be since we would be destructuring a hyphenated value?

> Should we mention in the readme example that we hard code an english value?

Definitely.

Reply:
@ryanseddon I added copy to the element example about expected override. But I don't think I can move `'aria-label'` into the destructured parameter list as it's an invalid property name.

Reply:
oh yeah good point